homeassistant/components/hyperion/strings.json

```diff
@@ -45,9 +45,10 @@
     "step": {
       "init": {
         "data": {
-          "priority": "Hyperion priority to use for colors and effects"
+          "priority": "Hyperion priority to use for colors and effects",
+          "effect_hide_list": "Hyperion effects to hide"
```

Comment:
In general, I think this PR looks really good. However, this part is confusing IMHO.

Have a multi-select to checkboxes for things you DON'T want. IMHO, that is the wrong way around.

Reply:
@frenck Thank you very much for the review! Just to confirm, before I rework this, you are suggesting that instead of an "effect_hide_list" we have an "effect_show_list"? That is, we offer a multi-select where a ticked box implies the effect IS visible. Is that what you mean?

The reason I did it this way around was actually to handle the empty case easily -- if there's no value specified for the option at all, I want to show all effects, i.e. the default behavior is inclusive -- no values present => no effects hidden. If I reverse it to effect_show_list, then I will need to also add special treatment for the option not being present at all, i.e.

 - Option not present at all (perhaps default, effect_show_list=None): Show all effects.
 - Option present, but empty list (effect_show_list=[]): Show no effects.
 - Option present, but partial list: Show those effects only.

So I think it's slightly more complicated to implement, but if you think that's clearer, more than happy to flip it around.

Let me know. Thanks!

Reply:
I don't think it has to be difficult. You can still store the "hide" list (as it's easier to process in code). This is about what is presented to the user.

So towards the user in the options, you can list all of them, based on the current effect list. On submit/store in the entry, you can inverse that result and store the "hide" list.

This way, you can keep almost all code logic and it will deal with new effects.